docs/docs/Support/troubleshooting.mdx

```diff
@@ -153,6 +202,15 @@ Clearing the cache erases your settings.
 If you want to retain your settings files, create a backup of those files before clearing the cache folder.
 :::
 
+### Langflow Desktop shows "latest version" but is outdated
+
+If Langflow Desktop shows you're on the latest version (for example, version `1.3.3`) but a newer version (like `1.5.0`) is available, this is likely because you're running a version prior to `1.4.2`.
+
+The automatic update feature through the graphical interface was introduced in version `1.4.2`.
+Versions prior to `1.4.2` don't include the necessary functionality to perform updates.
+
+To resolve, uninstall Langflow Desktop and download the latest version from the [official website](https://langflow.org/desktop).
```

Comment:
```suggestion
This happens because the automatic update feature in the UI was introduced in version 1.4.2.
Earlier versions can't automatically detect or apply updates.

To resolve this issue, uninstall Langflow Desktop, and then [download and install the latest version of Langflow Desktop](https://langflow.org/desktop).
```

Reply:
The original wording for the last sentence makes it sound like the docs site isn't official.